src/Features/Core/Portable/CodeLens/ReferenceMethodDescriptor.cs

```diff
@@ -11,6 +11,15 @@ namespace Microsoft.CodeAnalysis.CodeLens
     /// <summary>
     /// A caller method of a callee
     /// </summary>
+    /// <remarks>
+    /// Describe a caller method of a callee
+    /// </remarks>
+    /// <param name="fullName">Method's fully qualified name</param>
+    /// <param name="filePath">Method full path</param>
+    /// <remarks>
+    ///  Method full name is expected to be in the .NET full name type convention. That is,
+    ///  namespace/type is delimited by '.' and nested type is delimited by '+'
+    /// </remarks>
```

Comment:
📝 I'm not sure how multiple `<remarks>` elements are handled, but this should be fine for now.